open-metadata-implementation/view-services/tex-view/tex-view-api/build.gradle

```diff
@@ -1,17 +0,0 @@
-/*
```

Comment:
I notice the gradle file is deleted, yet the pom remains -- what is the intent?

Reply:
Apologies - I hadn'tt realised that delete had snuck its way into the PR. I deleted it as an experiment when I was having major trouble with IJ not being able to resolve various classes. I had not meant to delete it in the PR!

Reply:
I was adding a dependency and IJ wasn't resolving classes. I recalled our conversation about needing to manually tweak the gradle dependencies, so tried, failed and ended up removing the file :-)

Reply:
Have reinstated it in the latest push...